maps_dashboards/public/components/map_top_nav/top_nav_menu.tsx

```diff
@@ -119,12 +127,13 @@ export const MapTopNavMenu = ({
       showSaveQuery={true}
       showQueryInput={true}
       onQuerySubmit={handleQuerySubmit}
-      useDefaultBehaviors
```

Comment:
@junqiu-lei I removed this line, it turns out if `useDefaultBehaviors` is enabled, the callbacks such as `onRefresh` is not called. What's the intention of adding `useDefaultBehaviors`?

Reply:
Thanks @ruanyl, there didn't have intention, but yeah it looks like we need remove it to have onRefresh call.